lib/taskana-core/src/main/java/pro/taskana/task/internal/TaskServiceImpl.java

```diff
@@ -185,10 +185,14 @@ public Task cancelClaim(String taskId)
   @Override
   public Task forceCancelClaim(String taskId)
       throws TaskNotFoundException,
-          InvalidOwnerException,
-          NotAuthorizedOnWorkbasketException,
-          InvalidTaskStateException {
-    return this.cancelClaim(taskId, true);
+          InvalidTaskStateException,
+          InvalidOwnerException
```

Comment:
Please remove the `NotAuthorizedOnWorkbasketException` in the interface method

Reply:
Thanks for all the guidance! If I understand correctly, it is removed now.

Reply:
Somewhat. The Pipeline will fail since you forgot to remove the JavaDoc entry for that exception 😉 

Other then that it looks fine :)

Reply:
Ah yes, good catch, thanks